frontend/src/Editor/Components/FilePicker.jsx

```diff
@@ -241,18 +241,20 @@ export const FilePicker = ({
       });
       setSelectedFiles(fileData);
       onComponentOptionChanged(component, 'file', fileData);
-      onEvent('onFileSelected', { component }).then(() => {
-        setAccepted(true);
-        // eslint-disable-next-line no-unused-vars
-        return new Promise(function (resolve, reject) {
-          setTimeout(() => {
-            setShowSelectedFiles(true);
-            setAccepted(false);
-            onComponentOptionChanged(component, 'isParsing', false);
-            resolve();
-          }, 600);
-        });
-      });
+      onEvent('onFileSelected', { component })
+        .then(() => {
+          setAccepted(true);
+          // eslint-disable-next-line no-unused-vars
+          return new Promise(function (resolve, reject) {
+            setTimeout(() => {
+              setShowSelectedFiles(true);
+              setAccepted(false);
+              onComponentOptionChanged(component, 'isParsing', false);
+              resolve();
+            }, 600);
+          });
+        })
+        .then(() => onEvent('onFileLoaded ', { component }));
```

Comment:
There is an extra white space at the end of the **eventName**, 
`frontend/src/_helpers/appUtils.js :: 488-518` return `false` and the action is not been executed.